home-manager/home-manager

```diff
@@ -97,6 +97,46 @@ function setFlakeAttribute() {
     fi
 }
 
+function doInspectOption() {
+    setFlakeAttribute
+    if [[ -v FLAKE_CONFIG_URI ]]; then
+        errorEcho "Can't inspect options of a flake configuration"
+        exit 1
+    fi
+    setConfigFile
+    setHomeManagerNixPath
+
+    local extraArgs=("$@")
+
+    for p in "${EXTRA_NIX_PATH[@]}"; do
+        extraArgs=("${extraArgs[@]}" "-I" "$p")
+    done
+
+    if [[ -v VERBOSE ]]; then
+        extraArgs=("${extraArgs[@]}" "--show-trace")
+    fi
+
+    local HOME_MANAGER_CONFIG_NIX HOME_MANAGER_CONFIG_ATTRIBUTE_NIX
+    HOME_MANAGER_CONFIG_NIX=${HOME_MANAGER_CONFIG//'\'/'\\'}
+    HOME_MANAGER_CONFIG_NIX=${HOME_MANAGER_CONFIG_NIX//'"'/'\"'}
+    HOME_MANAGER_CONFIG_NIX=${HOME_MANAGER_CONFIG_NIX//$'\n'/$'\\n'}
+    HOME_MANAGER_CONFIG_ATTRIBUTE_NIX=${HOME_MANAGER_CONFIG_ATTRIBUTE//'\'/'\\'}
+    HOME_MANAGER_CONFIG_ATTRIBUTE_NIX=${HOME_MANAGER_CONFIG_ATTRIBUTE_NIX//'"'/'\"'}
+    HOME_MANAGER_CONFIG_ATTRIBUTE_NIX=${HOME_MANAGER_CONFIG_ATTRIBUTE_NIX//$'\n'/$'\\n'}
+    local modulesExpr
+    modulesExpr="let confPath = \"${HOME_MANAGER_CONFIG_NIX}\"; "
+    modulesExpr+="confAttr = \"${HOME_MANAGER_CONFIG_ATTRIBUTE_NIX}\"; in "
+    modulesExpr+="(import <home-manager/modules> {"
+    modulesExpr+=" configuration = if confAttr == \"\" then confPath else (import confPath).\${confAttr};"
+    modulesExpr+=" pkgs = import <nixpkgs> {}; check = true; })"
+
+    nixos-option \
```

Comment:
Cannot assume that nixos-option is installed. Unfortunately, I think it first has to be made available as a regular package in Nixpkgs so that we can pull it in as a dependency here.

Reply:
We can work around this for now thanks to `pkgs.path`.